clouddq/templates/dbt/dbt_project.yml

```diff
@@ -42,9 +42,13 @@ clean-targets: # directories to be removed by `dbt clean`
 models:
   data_quality:
     data_quality_engine:
-      materialized: table
+      +materialized: ephemeral
     rule_binding_views:
-      materialized: view
+      +materialized: view
+    entity_dq_statistics:
```

Comment:
Also here, after doing a few tests I realised that sometimes as a result of the execution (either test or normal run) I see the aggregated statistics object as table (with 6h expiration), but in other cases as a view (e.g. name dev:<project_name>.<dataset>__clouddq_integration_tests_input_data__ingestion_day_level__INVOICE_ID__NONE_1 ). I can see the env in the path name, would that depend on the dev? Or there are other factors?

Reply:
hmm the env should not appear in the path name. I'll ping you to double check this.

Reply:
Oh, yes, you're right. My mistake while copying - pasting multiline view_id, there's no ENV in the view name. But I still see a view instead of the table (with expiration), which I would expect..

Reply:
thanks for confirming! I wonder why this is being created as a view 🤔. technically all models in the `entity_dq_statistics` path should be materialized as a table. Could you share the file path tree of the resulting dbt folder?

Reply:
Confirmed, it's now creating a table (with expiration), I was using the old dbt config, it worked as expected after cleaning the dbt folder up.